scripts/install-cni.sh

```diff
@@ -121,15 +121,24 @@ if [ "$ENABLE_MASQUERADE" == "true" ]; then
   fi
 fi
 
-if [ "$ENABLE_PRIVATE_IPV6_ACCESS" == "true" ]; then
+if [ "$ENABLE_PRIVATE_IPV6_ACCESS" == "true" ] || [ "$ENABLE_IPV6" == "true" ]; then
   node_ipv6_addr=$(curl -s -k --fail "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/?recursive=true" -H "Metadata-Flavor: Google" | jq -r '.ipv6s[0]' ) ||:
 
   if [ -n "${node_ipv6_addr:-}" ] && [ "${node_ipv6_addr}" != "null" ]; then
     echo "Found nic0 IPv6 address ${node_ipv6_addr:-}. Filling IPv6 subnet and route..."
-    cni_spec=$(echo ${cni_spec:-} | sed -e \
-      "s#@ipv6SubnetOptional#, [{\"subnet\": \"${node_ipv6_addr:-}/112\"}]#g;
-       s#@ipv6RouteOptional#, {\"dst\": \"::/0\"}#g")
-    
+
+    if [ "$ENABLE_IPV6" == "true" ]; then
+      ipv6_subnet=${node_ipv6_addr%:*:*}:ffff:/112
+      echo "IPv6 subnet: ${ipv6_subnet:-}"
```

Comment:
This looks like a debug log. We should remove it.

Reply:
Done.